webpack.dist.config.js

```diff
@@ -5,7 +5,7 @@ const UglifyJsPlugin = require( 'uglifyjs-webpack-plugin' );
 const LodashModuleReplacementPlugin = require( 'lodash-webpack-plugin' );
 
 module.exports = {
-	entry: './index.js',
+	entry: './dist/index.js',
```

Comment:
What's the difference between `/dist/index` and `/targets/dist/index`? The naming might be a bit confusing.

Reply:
It was a copy paste error when splitting in smaller PRs. This is fixed in a future PR.

Reply:
doesn't this need to be `targets/dist/index` for it to work? `npm run targets:dist` fails otherwise.

Reply:
Like I mentioned above, this was a typo and it is fixed in a PR built on top of this. I'll push the fix here also.

Reply:
Ah, I thought you wanted to merge small PRs as they were ready, but I guess you want to build a mega-PR and only commit when everything is ready.

Reply:
The issue was that most of them are dependent on other updates that is why I chose to built them one on top of another. These PRs are mostly ground work but this shouldn't be the case for future work.